config.rb

```diff
@@ -71,6 +74,10 @@ def initialize(app, options_hash={}, &block)
 # Helpers
 # ----------------------------------------------
 helpers do
+  # turn string to date object for comparison
+  def d(string)
```

Comment:
Is it okay to name it date? we use it very often, to name it d depends on the people that they will develop it.
Vorschlaege?